src/containers/EditListingPage/EditListingWizard/EditListingPricingPanel/EditListingPricingPanel.js

```diff
@@ -42,10 +49,22 @@ const EditListingPricingPanel = props => {
   const classes = classNames(rootClassName || css.root, className);
   const initialValues = getInitialValues(props);
   const isPublished = listing?.id && listing?.attributes?.state !== LISTING_STATE_DRAFT;
-  const priceCurrencyValid =
-    marketplaceCurrency && initialValues.price instanceof Money
-      ? initialValues.price.currency === marketplaceCurrency
-      : !!marketplaceCurrency;
+
+  const publicData = listing?.attributes?.publicData;
+  const listingTypeConfig = getListingTypeConfig(publicData, listingTypes);
+  const transactionProcessAlias = listingTypeConfig.transactionType.alias;
+
+  const isStripeCompatibleCurrency = isValidCurrencyForTransactionProcess(
+    transactionProcessAlias,
+    marketplaceCurrency,
+    transactionProcessAlias === 'default-inquiry/release-1' ? null : 'stripe'
```

Comment:
Omit this 3rd parameter. Your function should work without it. (And change the variable name. E.g. ” isStripeCompatibleCurrency” > ” isCompatibleCurrency”